Cognigy/WebchatWidget
# Trap focus within chat window

Current behavior: During Tab/ Shift + Tab navigation, focus can be moved to elements outside the chat window. 
Expected behavior: When enableFocusTrap is set to true, focus should never leave the chat window. By default, enableFocusTrap is false, so focus can leave the chat window.

To test: 
- Add webchat setting `enableFocusTrap: true`
- Navigate using keyboard and see is the focus does not leave the chat. The focus order should also be logical during both Tab and Shift + Tab navigation
- Also test by setting enableFocusTrap to false. There should not be any change in the behavior
